belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/api/InputValidationIssues.java

```diff
@@ -107,9 +107,33 @@ public static <T extends Comparable<T>> InputValidationIssue outOfRange(InEnum i
     }
 
     public static InputValidationIssue referencedResourceNotFound(InEnum in, String name, Object value) {
+        return referencedResourceNotFound(in, name, name, value);
+    }
+
+    public static InputValidationIssue referencedResourceNotFound(InEnum in, String parameterName, String resourceName,
```

Comment:
Should we also add an overload with the resource name parameter for the [6 refData methods](https://github.com/belgif/rest-problem-java/blob/cb8009fb4a2c723fb0e942b30a8402df187bf230/belgif-rest-problem-validator/src/main/java/io/github/belgif/rest/problem/validation/AbstractRequestValidator.java#L281-L373) in AbstractRequestValidator? Maybe not... going to 12 methods is a bit much.